sympy/sets/tests/test_sets.py

```diff
@@ -941,3 +941,7 @@ def test_issue_10113():
 def test_issue_10248():
     assert list(Intersection(S.Reals, FiniteSet(x))) == [
         And(x < oo, x > -oo)]
+
+
+def test_issue_10337():
+    raises(TypeError, lambda: FiniteSet(2)<3)
```

Comment:
Better add the others for le, gt, ge and also ones for eq and ne showing that an answer should be obtained of False and True, respectively.